files/galaxy/config/activation-email.html

```diff
@@ -23,36 +23,86 @@
 
-    <!-- Load custom font from Google Fonts -->
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
-  </head>
-  <body style="font-family: 'Roboto', sans-serif;">
+<head>
+  <meta charset="utf-8">
+  <title>Galaxy account activation</title>
 
-    <p style="font-size: 14pt;">
-      Hello {{ name }},<br><br>
+  <!-- Load custom font from Google Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
 
-      Thank you for registering an account with Galaxy Australia.
+<body style="font-family: 'Roboto', sans-serif;">
 
-      To finalise registration, please click on the following link to verify your email address: 
-      <a href="{{ activation_url }}">click here</a>
+  <h3 style="font-size: 18pt;">Thanks for signing up to use Galaxy Australia!</h3>
 
-      By registering an account you agree to abide by <a href="{{ terms_url }}">Galaxy Australia's Terms of Service</a>.
+  <p style="font-size: 14pt;">
+    Hello {{ name }},<br><br>
 
-      Please contact us if you need help with your account at: <a href="mailto:{{ contact_email }}">{{ contact_email }}</a>.
+    In order to complete the activation process for {{ user_email }} (begun on
+    {{ date }} at {{ hostname }}), please
+    <a href="{{ activation_url }}">click here</a>
+    to verify your account.
+    <br><br>
 
-      For security reasons, the account activation link will expire in {{ expiry_days }} days.
-    </p>
+    By clicking on the above link and opening a Galaxy Australia account you are also
+    confirming that you have read and agreed to
+    {% if terms_url %}
+    <a href="{{ terms_url }}">Galaxy Australia's Terms and Conditions</a>
+    {% else %}
+    Galaxy Australia's Terms and Conditions
+    {% endif %}
+    for use of this service. This includes a data storage limit per user
+    account. Attempts to subvert this limit by creating multiple accounts or
+    through any other method may result in termination of all associated
+    accounts and data.
+    <br><br>
 
-    <p style="font-size: 14pt;">
-      Regards,<br>
-      Galaxy Australia Team
-    </p>
+  <ul style="font-size: 14pt;">
+    {% if contact_email %}
+    <li>
+      If you need help with your account, please contact us at
+      <a href="mailto:{{ contact_email }}">{{ contact_email }}</a>
+    </li>
+    {% endif %}
+    {% if instance_resource_url %}
+    <li>
+      Browse available resources:
+      <a href="{{ instance_resource_url }}">{{ instance_resource_url }}</a>
+    </li>
+    {% endif %}
+    <li>
+      Find out more about the Galaxy Project at
+      <a href="galaxyproject.org">galaxyproject.org</a>.
```

Comment:
I should issue a patch to fix this in the default template. Can you change href to include `https://`?